### PR TITLE
5.0.2 Release

### PR DESCRIPTION
**Pull request by QuantumNightmare**
### Changelog

**Bug fix**
* When logging a null exception, the stack trace included in the Crash Report sent to Raygun will now correctly be the code execution stack up to the point where the log statement was made. This is functionality that was added in the previous version, but was erroneously using the code execution stack of a background thread.

### CODE REVIEW

**Comment by QuantumNightmare** [comment]
@nblumhardt When you get a chance, please review (and merge if approved) this new release that contains a single bug fix. As you'll see, the build script has failed due to being unable to push the dev build to NuGet.

File: src/Serilog.Sinks.Raygun/Sinks/Raygun/NullException.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace Serilog.Sinks.Raygun.Sinks.Raygun
```
Comment (nblumhardt) [inline src/Serilog.Sinks.Raygun/Sinks/Raygun/NullException.cs:4]: Nit - namespace is unhappy here; adding `<RootNamespace>Serilog</RootNamespace>` to the CSPROJ should prevent these popping up in future

**Review by nblumhardt — APPROVED** [review]
LGTM; publishing key is now updated, let me know if you hit any more difficulties :-)

**Comment by QuantumNightmare** [comment]
Thanks @nblumhardt, I've made the namespace change and pushed the new 5.0.2 version with this bug fix. I needed to edit the release from "v88" to "v5.0.2", so not sure if that's normal. I also changed the version of the dev branch to 5.0.3, but this pushed a pre release version to NuGet. Should that have been committed straight to the dev branch instead of through a PR?

**Comment by nblumhardt** [comment]
Direct to `dev` and a PR have the same effect - I think appending `[skip ci]` to the commit message sometimes/usually avoids publishing anything.

**Comment by QuantumNightmare** [comment]
Yeah, I did have `[skip ci]` in the commit to my branch at least. Maybe it needed to be in the PR name too in order to become part of the "Merge pull request" commit too.

**Pull request merged**